doc/source/working_with_kiwi/legacy_kiwi.rst

```diff
@@ -1,7 +1,7 @@
 .. _legacy_kiwi:
 
-Legacy KIWI vs. Next Generation
-===============================
+Legacy KIWI vs. {kiwi-product}
+==============================================
```

Comment:
You could remove some of the equal signs. :wink:

Reply:
no you can't. {kiwi-product} is replaced with content prior the sphinx rendering. The content string is longer than the macro. If you delete the signs you break the rendering